index.rst

```diff
@@ -297,40 +297,14 @@ Full Table of Contents
 .. toctree::
    :maxdepth: 3
 
-   setup
-   help
-   pullrequest
-   runtests
-   coverage
-   docquality
-   documenting
-   silencewarnings
-   fixingissues
-   tracker
-   triaging
-   communication
-   porting
-   coredev
-   developers
-   committing
-   devcycle
-   buildbots
-   stdlibchanges
-   langchanges
-   experts
-   gdb
-   exploring
-   grammar
-   parser
-   compiler
-   garbage_collector
-   extensions
-   c-api
-   coverity
-   clang
-   buildworker
-   motivations
-   gitbootcamp
+   starting/index
+   developing/index
+   triaging/index
+   documenting/index
+   testing/index
+   committing/index
+   exploring/index
+   tooling/index
```

Comment:
> I admire the parallelism between these (though `tooling` is a false friend to the others)

I tried... tooling was the hardest and I wasn't that happy with it. I'll do the rename but to preserve a good history I'll rebase and force push with it after all your other comments.

A

Reply:
Yeah, I felt really bad since I knew you must have put a good deal of thought into this, and I've tried for the exact same sort of thing in the past and been really disappointed if I couldn't get it to work out.